src/components/modules/weeklyBanner/WeeklyBanner.tsx

```diff
@@ -0,0 +1,323 @@
+import { css } from 'emotion';
+import { ThemeProvider } from 'emotion-theming';
+import { body, headline, textSans } from '@guardian/src-foundations/typography/cjs';
+import { neutral } from '@guardian/src-foundations/palette';
+import { from, until } from '@guardian/src-foundations/mq';
+import { space } from '@guardian/src-foundations';
+import { Button, LinkButton, buttonReaderRevenue } from '@guardian/src-button';
+import { brand } from '@guardian/src-foundations/themes';
+import { SvgGuardianLogo } from '@guardian/src-brand';
+import { SvgClose } from '@guardian/src-icons';
+
+const banner = css`
+    html {
+        box-sizing: border-box;
+    }
+    *,
+    *:before,
+    *:after {
+        box-sizing: inherit;
+    }
+    box-sizing: border-box;
+    display: flex;
+    justify-content: center;
+    width: 100%;
+    background-color: #3f464a;
+    color: ${neutral[100]};
+`;
+
+const contentContainer = css`
+    box-sizing: border-box;
+    display: flex;
+    flex-direction: column;
+    max-width: 100%;
+    ${from.tablet} {
+        display: flex;
+        flex-direction: row;
+    }
+    ${from.wide} {
+        max-width: 1250px;
+    }
+`;
+
+const topLeftComponent = css`
+    width: 100%;
+    padding: ${space[4]}px ${space[4]}px 0 ${space[4]}px;
+    button {
+        margin-left: ${space[3]}px;
+    }
+    ${from.tablet} {
+        padding: ${space[4]}px;
+        width: 65%;
+    }
+
+    ${from.desktop} {
+        width: 50%;
+    }
+
+    ${from.wide} {
+        width: 53%;
+    }
+`;
+
+const heading = css`
+    ${headline.small({ fontWeight: 'bold' })};
+    margin: 0;
+    max-width: 100%;
+
+    @media (max-width: 740px) {
+        max-width: 85%;
+    }
+
+    ${until.mobileLandscape} {
+        ${headline.xsmall({ fontWeight: 'bold' })};
+    }
+`;
+
+const paragraph = css`
+    ${body.medium()}
+    line-height: 135%;
+    margin: ${space[2]}px 0 ${space[6]}px;
+    max-width: 85%;
+    ${from.tablet} {
+        max-width: 100%;
+    }
+    ${from.desktop} {
+        font-size: 20px;
+        margin: ${space[3]}px 0 ${space[9]}px;
+    }
+`;
+
+const buttonTextDesktop = css`
+    display: none;
+    ${from.desktop} {
+        display: block;
+    }
+`;
+
+const buttonTextMobileTablet = css`
+    display: block;
+    ${from.desktop} {
+        display: none;
+    }
+`;
+
+const siteMessage = css`
+    margin: ${space[3]}px 0 ${space[4]}px;
+    ${textSans.small()};
+    color: ${neutral[100]};
+    a,
+    :visited {
+        color: ${neutral[100]};
+        font-weight: bold;
+    }
+`;
+
+const bottomRightComponent = css`
+    max-height: 215px;
+    overflow: hidden;
+    margin-top: -25px;
+
+    ${from.mobileMedium} {
+        max-height: 280px;
+        margin-top: -15px;
+    }
+
+    ${from.tablet} {
+        display: flex;
+        align-items: flex-end;
+        margin-top: 0;
+        max-width: 50%;
+        max-height: 100%;
+    }
+
+    ${from.desktop} {
+        display: flex;
```

Comment:
Don't need `display: flex` here it's defined at tablet breakpoint

Reply:
👍